content/en/dashboards/functions/rate.md

```diff
@@ -3,6 +3,10 @@ title: Rate
 kind: documentation
 aliases:
     - /graphing/functions/rate/
+further_reading:
+- link: "/monitors/guide/alert-on-no-change-in-value/"
+  tag: "Guide"
+  text: "Alert on no change in value"
 ---
 
```

Comment:
Can we add an Overview section that says "this page describes the rate function"?

Reply:
Thanks Austin, that makes sense. I'm planning on restructuring all the function pages. When I outline a template for them, I'll add an overview section for each page! Or would you recommend adding the section anyway?